### PR TITLE
Do/exclude users forward forms

### PR DESCRIPTION
**Pull request by dominicochoa-dimagi**
## Technical Summary

[Jira ticket](https://dimagi-dev.atlassian.net/browse/USH-1080?atlOrigin=eyJpIjoiZTI3ZTBjNTA1YzQyNDZhMGIzMjNhZGY5NzgxNjY2MGMiLCJwIjoiaiJ9)

Took the code from the case forwarding code block and repurposed it for the form repeater.

## Feature Flag

Data Forwarding

### QA Plan

None

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/motech/repeaters/forms.py [hunk 1]
```diff
@@ -102,11 +102,27 @@ class FormRepeaterForm(GenericRepeaterForm):
         label="Include 'app_id' URL query parameter.",
         initial=True
     )
+    black_listed_users = forms.MultipleChoiceField(
+        required=False,
+        label=_('Users to exclude'),
+        widget=forms.SelectMultiple(attrs={'class': 'hqwebapp-select2'}),
+        help_text=_('Case creations and updates submitted by these users will not be forwarded')
```
Comment (snopoke) [inline corehq/motech/repeaters/forms.py:109]:
```suggestion
        help_text=_('Forms submitted by these users will not be forwarded')
```

File: corehq/motech/repeaters/forms.py [hunk 2]
```diff
@@ -102,11 +102,27 @@ class FormRepeaterForm(GenericRepeaterForm):
         label="Include 'app_id' URL query parameter.",
         initial=True
     )
+    black_listed_users = forms.MultipleChoiceField(
```
Comment (snopoke) [inline corehq/motech/repeaters/forms.py:105]: nit: rename 'blacklist' to 'blocklist'
```suggestion
    user_blocklist = forms.MultipleChoiceField(
```

**Review by snopoke — COMMENTED** [review]
Looks good pending some minor changes.

**Review by kaapstorm — COMMENTED** [review]
In addition to Simon's suggestions, a "pro tip" on Dimagi conventions: Feel free to rebase if nobody has reviewed your PR yet. (If your PR has been reviewed, then rebasing makes the review feedback difficult to follow.)

I mention this because the last 3 commits could use `git rebase master --interactive` + "squash", or `git commit --fixup` + `git rebase master --autosquash` to keep your branch clean :nerd_face: and quick to review.

File: corehq/motech/repeaters/forms.py [hunk 3]
```diff
@@ -120,7 +136,7 @@ class CaseRepeaterForm(GenericRepeaterForm):
         required=False,
         label=_('Users to exclude'),
         widget=forms.SelectMultiple(attrs={'class': 'hqwebapp-select2'}),
-        help_text=_('Case creations and updates submitted by these users will not be forwarded')
+        help_text=_('Form creations and updates submitted by these users will not be forwarded')
```
Comment (snopoke) [inline corehq/motech/repeaters/forms.py:139]:
```suggestion
        help_text=_('Case creations and updates submitted by these users will not be forwarded')
```

File: corehq/motech/repeaters/views/repeaters.py [hunk 4]
```diff
@@ -249,6 +249,8 @@ def set_repeater_attr(self, repeater, cleaned_data):
         repeater = super().set_repeater_attr(repeater, cleaned_data)
         repeater.include_app_id_param = (
             self.add_repeater_form.cleaned_data['include_app_id_param'])
+        repeater.black_listed_users = (
```
Comment (snopoke) [inline corehq/motech/repeaters/views/repeaters.py:252]:
```suggestion
        repeater.user_blocklist = (
```

**Review by snopoke — COMMENTED** [review]
Almost there!

**Review by snopoke — APPROVED** [review]
🚀

**Pull request merged**